client/stylesheets/_toCat.scss

```diff
@@ -3142,11 +3142,11 @@ i:hover {
 }
 
 .homeCardImg {
-    height: 200px !important;
+    height: 100% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces